### PR TITLE
Update landing page

### LINKED ISSUE #243: improve dashboard landing page
Similar to, https://github.com/SolarArbiter/SolarArbiter.github.io/issues/145, we need some better hints at workflow on the dashboard home page. 

Also true for the non-authenticated page. It doesn't need to be a lot, but it should make people want to sign up.

### PR DESCRIPTION
**Pull request by lboeman**
closes #243 
Mostly touched the unauthenticated landing page, which i think is an improvement, and added links and reworded some of the authenticated landing page. Including screenshots in case they're easier to read than the html. 

Authenticated:
authenticated

Unauthenticated:
unauthenticated

### CODE REVIEW

File: sfa_dash/templates/index.html [hunk 1]
```diff
@@ -6,33 +6,58 @@
...
+link lists Aggregates available to you. Use this menu to
+<a  href="https://solarforecastarbiter.org/documentation/dashboard/working-with-data/#create-new-aggregate">create new aggregates</a>. Selecting an aggregate allows for
+<a href="https://solarforecastarbiter.org/documentation/dashboard/working-with-data/#add-observations-to-an-aggregate">associating existing observations with an aggregate</a>
+and viewing aggregated data.</p>
 
-<p>The <b>Aggregates</b> link lists Aggregates available to you. From
-the aggregates menu, you will be able to create new aggregates, associate
-existing observations with an aggregate and view aggregate data.</p>
+<p>The <a class="font-weight-bold" href="/reports" >Reports</a>
+link lists Reports available to you and allow you to create new reports.</p>
```
Comment (wholmgren) [inline sfa_dash/templates/index.html:51]:
```suggestion
link lists Reports available to you and allows you to create new reports.</p>
```

**Review by wholmgren — COMMENTED** [review]
Definitely an improvement for the unauthenticated page.

One concern about the authenticated page: the create options are not available to all users, so the text might cause some confusion.

File: sfa_dash/templates/index.html [hunk 2]
```diff
@@ -6,33 +6,58 @@
 <h2 class="page-title">Solar Forecast Arbiter Dashboard</h2>
 {% include "sections/notifications.html" %}
 {% if current_user is none %}
-<p>Welcome to the Solar Forecast Arbiter Dashboard. If you have an account, please
+<p>Welcome to the Solar Forecast Arbiter, a tool for evaluating your
+solar power and irradiance forecasts. The Arbiter will allow you to securely
+upload forecast data for your own private evaluation, or to participate in
+competitive forecast trials that compare forecast performance between
+anonymous participantss.</p>
```
Comment (wholmgren) [inline sfa_dash/templates/index.html:13]:
```suggestion
anonymous participants.</p>
```

**Review by wholmgren — COMMENTED** [review]
here are some quick fixes. I want to come back and tweak another sentence in a bit

File: sfa_dash/templates/index.html [hunk 3]
```diff
@@ -6,33 +6,58 @@
 <h2 class="page-title">Solar Forecast Arbiter Dashboard</h2>
 {% include "sections/notifications.html" %}
 {% if current_user is none %}
-<p>Welcome to the Solar Forecast Arbiter Dashboard. If you have an account, please
+<p>Welcome to the Solar Forecast Arbiter, a tool for evaluating your
+solar power and irradiance forecasts. The Arbiter will allow you to securely
+upload forecast data for your own private evaluation, or to participate in
+competitive forecast trials that compare forecast performance between
+anonymous participants.</p>
```
Comment (wholmgren) [inline sfa_dash/templates/index.html:13]: can you edit and html-ify the following?

The Arbiter allows you to:

* Securely upload forecast and observation data
* Analyze historical forecast accuracy and track ongoing forecast accuracy
* Optionally share data and analysis reports with other users
* Participate in anonymous, multi-vendor, operational forecast trials

**Review by wholmgren — APPROVED** [review]

**Comment by lboeman** [comment]
Added mention of getting the admin section through the *Account* menu, and added a link to the administration docs on the admin landing page. 
I also reworded the authenticated landing to first mention general features that a new user might have access to and then mention the features accessible with the correct permissions. 
@wholmgren ?
Screenshot from 2020-11-17 10-20-34

**Review by wholmgren — APPROVED** [review]
@alorenzo175 can you give this a quick look for typos and anything else you think is unclear? I don't want to get this one wrong.

File: sfa_dash/templates/index.html [hunk 4]
```diff
@@ -6,33 +6,73 @@
...
+or <a  href="https://solarforecastarbiter.org/documentation/dashboard/working-with-data/#create-new-observation-or-forecast">create Observations, Forecasts, and Probabilistic Forecasts</a>
+at existing Sites in this menu.</p>
+
+<p>The <a class="font-weight-bold" href="/aggregates" >Aggregates</a>
+link lists Aggregates available to you. Use this menu to view existing
+Aggregates and associated data. Selecting an Aggregate displays its constituent
+Observations and aggregated observation data. Users with the proper permissions
+can <a href="https://solarforecastarbiter.org/documentation/dashboard/working-with-data/#create-new-aggregate">create new Aggregates</a>,
+<a href="https://solarforecastarbiter.org/documentation/dashboard/working-with-data/#add-observations-to-an-aggregate">associate existing Observations with an Aggregate</a>,
+and <a href="https://solarforecastarbiter.org/documentation/dashboard/working-with-data/#create-new-forecast">create Forecasts, and Probabilsitic Forecasts</a>
```
Comment (alorenzo175) [inline sfa_dash/templates/index.html:57]:
```suggestion
or <a href="https://solarforecastarbiter.org/documentation/dashboard/working-with-data/#create-new-forecast">create Forecasts and Probabilistic Forecasts</a>
```

**Review by alorenzo175 — APPROVED** [review]

**Pull request merged**